#! /bin/sh /usr/share/dpatch/dpatch-run
## qpsmtpd-0.30rc2.smtp-loglevel.dpatch by  <devin@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Log SMTP conversation at LOGINFO rather than LOGDEBUG

@DPATCH@

diff -aruN qpsmtpd-0.30rc2.aux-groups/lib/Qpsmtpd/TcpServer.pm qpsmtpd-0.30rc2.smtp-loglevel/lib/Qpsmtpd/TcpServer.pm
--- qpsmtpd-0.30rc2.aux-groups/lib/Qpsmtpd/TcpServer.pm	2005-01-29 21:40:24.000000000 -0800
+++ qpsmtpd-0.30rc2.smtp-loglevel/lib/Qpsmtpd/TcpServer.pm	2005-07-25 01:44:52.000000000 -0700
@@ -61,7 +61,7 @@
   while (<STDIN>) {
     alarm 0;
     $_ =~ s/\r?\n$//s; # advanced chomp
-    $self->log(LOGDEBUG, "dispatching $_");
+    $self->log(LOGINFO, "dispatching $_");
     $self->connection->notes('original_string', $_);
     defined $self->dispatch(split / +/, $_)
       or $self->respond(502, "command unrecognized: '$_'");
@@ -74,7 +74,7 @@
   my ($self, $code, @messages) = @_;
   while (my $msg = shift @messages) {
     my $line = $code . (@messages?"-":" ").$msg;
-    $self->log(LOGDEBUG, $line);
+    $self->log(LOGINFO, $line);
     print "$line\r\n" or ($self->log(LOGERROR, "Could not print [$line]: $!"), return 0);
   }
   return 1;
